examples/contact-sensor-app/nxp/k32w1/README.md

```diff
@@ -207,6 +210,8 @@ chip_subscription_timeout_resumption: same as above + try to re-establish timeou
 using Fibonacci backoff for retries pacing.
 ```
 
+`chip_enable_icd_dsls` may be used for enabling DSLS (Dynamic SIT LIT Support).
```

Comment:
Is there somewhere in the main SDK where the DSLS acronym can be defined formally?
(Specifically the name "Dynamic SIT LIT Support")
Adding a comment to the definition of CHIP_CONFIG_ENABLE_ICD_DSLS may be good, but that is defined in an automated way in a BUILD.gn file.

Reply:
chip_enable_icd_dsls is defined in src/app/icd/icd.gni and I added a comment there. Do you see a better place?

Reply:
let's move this offline for the better place. thanks